docs/X64WINE.md

```diff
@@ -122,6 +123,8 @@ This install method allows you to install different versions of Wine64/Wine.  Yo
 	sudo ln -s ~/wine/bin/wineserver /usr/local/bin/wineserver
 	sudo chmod +x /usr/local/bin/wine /usr/local/bin/wine64 /usr/local/bin/wineboot /usr/local/bin/winecfg /usr/local/bin/wineserver
 ```
+## Installing the needed shared libraries (Optional)
+Copy the shared libraries folder (`x64lib` or `x86lib`) from the Box64 repository and transfer it to the target device. Ensure that the shared libraries are located in the appropriate search paths. Otherwise, errors may occur because some libraries lack a wrapper and must be emulated. This means their dependency libraries need to exist in the x86-64 architecture format.
```

Comment:
I don't understand this. x86_64 and x86 libraries in `x64lib` and `x86lib` will be automatically copied to `/usr/lib/box64-x86_64-linux-gnu/` and `/usr/lib/box64-i386-linux-gnu/` if you do `sudo make install` after the compiling.

Reply:
Sorry for the confusion. I mean that the situation is where box64 was compiled using a cross-compiler, so there is only the box64 binary in the environment.

Reply:
Additional installation steps may be necessary when copying only the `box64` executable file without running `make install` in cross-build environments.

Reply:
In that case, it has nothing to do with wine.

Reply:
I would suggest that this goes into the COMPILE.md maybe? With a new section for cross-compiling.

Reply:
> I would suggest that this goes into the COMPILE.md maybe? With a new section for cross-compiling and manual installation.

Agree. That would be informative.
@devarajabc, you should refine the subject of this pull request accordingly.